cmd/kerberos.go

```diff
@@ -0,0 +1,236 @@
+package cmd
+
```

Comment:
No comments on what this controller does

Reply:
It control kerberos! 
Yeah I agree that docs/comments can be improved.
I will go improve the docs/comments for this controller